src/sage/matroids/dual_matroid.py

```diff
@@ -487,17 +487,17 @@ def __ne__(self, other):
         EXAMPLES::
 
             sage: from sage.matroids.advanced import *
-            sage: M1 = matroids.named_matroids.Fano()                                   # optional - sage.rings.finite_rings
-            sage: M2 = CircuitClosuresMatroid(M1.dual())                                # optional - sage.rings.finite_rings
-            sage: M3 = CircuitClosuresMatroid(M1).dual()                                # optional - sage.rings.finite_rings
+            sage: M1 = matroids.named_matroids.Fano()
+            sage: M2 = CircuitClosuresMatroid(M1.dual())
+            sage: M3 = CircuitClosuresMatroid(M1).dual()
             sage: M4 = CircuitClosuresMatroid(groundset='abcdefg',
             ....:   circuit_closures={3: ['abcdefg'], 2: ['beg', 'cdb', 'cfg',
             ....:                'ace', 'fed', 'gad', 'fab']}).dual()
-            sage: M1.dual() != M2  # indirect doctest                                   # optional - sage.rings.finite_rings
```

Comment:
Why removing the `indirect doctest` statement ? Isn't it useful ?

Reply:
Thanks for catching this mistake. I've restored it in 4d112691ff